src/content/resources/whats-new.md

```diff
@@ -18,6 +18,53 @@ and follow the [Dart blog][].
 [dart-announce]: https://groups.google.com/a/dartlang.org/d/forum/announce
 [Dart blog]: https://blog.dart.dev
 
+## 3.10 release
+
+_Released on: November 12, 2025_
+
+This section lists notable changes made from August 14, 2025,
+through November 12, 2025 to [dart.dev](https://dart.dev/docs).
+For details about the 3.10 release of Dart,
+check out the [3.10 announcement][] and the
+[3.10 SDK changelog][3-10-changelog].
+
+### Documentation updates {:.no_toc}
+
+*   Added a guide for the new [dot shorthands][] feature.
+*   Added a guide for the new [analyzer plugin][] feature.
+*   Added a guide for the [`dart build`][] tool.
+*   Added a guide for the [`dart install`][] tool.
+*   Extended the [`@Deprecated` annotation][] content with new features.
```

Comment:
Can we be a more specific here? It's not a new feature of the pre-existing `@Deprecated` annotation, but new annotations for deprecating other capabilities.

```suggestion
*   Documented the new [`@Deprecated` annotation][] varieties.
```